packages/commonwealth/server/models/contract_abi.ts

```diff
@@ -0,0 +1,43 @@
+import * as Sequelize from 'sequelize'; // must use "* as" to avoid scope errors
+import { DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+
+export type ContractAbiAttributes = {
+  id: number;
+  abi: string;
+};
+
+export type ContractAbiInstance = ModelInstance<ContractAbiAttributes> & {
+    // add mixins as needed
+};
+
+export type ContractAbiModelStatic = ModelStatic<ContractAbiInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): ContractAbiModelStatic => {
+  const ContractAbi = <ContractAbiModelStatic>sequelize.define(
+    'ContractAbi',
+    {
+      id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+      abi: { type: dataTypes.JSONB, allowNull: false, unique: true },
+      created_at: { type: dataTypes.DATE, allowNull: false },
```

Comment:
We may want to specify the source (etherscan vs user-provided).

Reply:
should we just have a `verified` column then? and we can periodically update + verify against etherscan accordingly?

Reply:
reasonable solution, unless we anticipate having different verified / user-submitted ABIs